docs/animations/ScrollViewerExtensions.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
We also have ScrollViewerExtensions in the main package, so do we want this in the extensions folder still anyway?

Reply:
They're two separate classes from two different assemblies, I was thinking using the same docs page could've been confusing? As in, if people tried to only reference one of the two while still expecting to see all the extensions 🤔